mock-core-configs/etc/mock/opensuse-tumbleweed-aarch64.cfg

```
@@ -8,6 +8,7 @@ config_opts['useradd'] = '/usr/sbin/useradd -o -m -u %(uid)s -g %(gid)s -d %(hom
 config_opts['releasever'] = '0'
 config_opts['macros']['%dist'] = '.suse.tw%(sh -c ". /etc/os-release; echo \$VERSION_ID")'
 config_opts['package_manager'] = 'dnf'
+config_opts['dnf_install_command'] = 'install dnf dnf-plugins-core shadow-utils distribution-gpg-keys'
```

The problem of tumbleweed is that there's no `shadow-utils` package, there's `shadow` package (which is installed automatically, even when it is not in `dnf_install_command`).

---

Oh, got it now. So solution should be that I just simply remove shadow-utils from this `dnf_install_command`and it will take precedence over `dnf_install_command` in `util.py`. So it won't fail on no package found with this name. Am I right?  

```suggestion
config_opts['dnf_install_command'] = 'install dnf dnf-plugins-core distribution-gpg-keys'
```

---

That should be good enough.